conv_rnn/data.py

```diff
@@ -0,0 +1,48 @@
+import numpy as np
+import os
+import re
+
+def sst_tokenize(sentence):
+    extraneous_pattern = re.compile(r"^(--lrb--|--rrb--|``|''|--|\.)$")
+    words = []
+    for word in sentence.split():
+        if re.match(extraneous_pattern, word):
+            continue
+        words.append(word)
+    return words
+
+class SSTDataLoader(object):
```

Comment:
Please take a look at http://pytorch.org/docs/master/data.html#torch.utils.data.Dataset and http://pytorch.org/docs/master/data.html#torch.utils.data.Dataset. These are PyTorch utility classes that can handle things like data shuffling, batching of different size easily so you don't have to manually deal with them in your code.